ocs_ci/deployment/vmware.py

```diff
@@ -283,6 +283,10 @@ def post_destroy_checks(self):
                 f"cluster {self.cluster}"
             )
 
+        # check whether folder exists in templates
+        if self.vsphere.is_folder_exists(pool, self.cluster, self.datacenter):
```

Comment:
I'm not sure of any other usages of `destroy_folder` but would it make sense to perform this check within `destroy_folder` instead? This way we won't need to do this check before calling it if we have to call it in other places. This is of course assuming we wouldn't ever want to call `destroy_folder` on a folder that doesn't exist. 

I'd also like to see a log message saying that we couldn't find the folder if deletion is skipped due to that reason.

Reply:
Done